plugins/modules/source_control/git/git_acp.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Cumulus Networks <ce-ceng@cumulusnetworks.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: git_acp
+author:
+    - "Federico Olivieri (@Federico87)"
+version_added: "2.10"
+short_description: Perform git commiit and/or git push perations.
+description:
+    - Manage git commits and git push on local git directory
+options:
+    folder_path:
+        description:
+            - full folder path where .git/ is located.
+        required: true
+        type: str
+    user:
+        description:
+            - git username for https operations.
+        type: str
+    token:
+        description:
+            - git API token for https operations.
+        type: str
+    comment:
+        description:
+            - git commit comment. Same as "git commit -m".
+        type: str
+        required: true
+    add:
+        description:
+            - list of files to be staged. Same as "git add ."
+              Asterisx values not accepted. i.e. "./*" or "*".
+        type: list
+        default: ["."]
+        elements: str
+    branch:
+        description:
+            - git branch where perform git push.
+        required: True
+        type: str
+    push_option:
+        description:
+            - git push options. Same as "git --push-option=option".
+        type: str
+    mode:
+        description:
+            - git operations are performend eithr over ssh channel or https.
+              Same as "git@git..." or "https://user:token@git..." or "git init --bare"
+        choices: ['ssh', 'https', 'local']
+        default: ssh
+        type: str
+    url:
+        description:
+            - git repo URL. If not provided, the module will use the same mode used by "git clone".
+        type: str
+requirements:
+    - git>=2.10.0 (the command line tool)
+'''
+
+EXAMPLES = '''
+
+# Commit and push changes via https.
+- git_acp:
```

Comment:
For new modules, we need to use the Fully Qualified Collection Name (FQCN)
Also, it's considered good practice to provide a `name:`

```suggestion
- name: Commit and push changes via https.
  community.general.git_acp:
```

Same applies to other examples